src/test/java/org/broadinstitute/hellbender/utils/haplotype/HaplotypeBAMWriterUnitTest.java

```diff
@@ -180,6 +175,7 @@ public void testCreateSAMFromHeader(
                     readLikelihoods);
         }
 
+        Assert.assertEquals(mockDest.getBAMOutputHeader().getReadGroups().size(), 1);
```

Comment:
Since you already added a method below to get the read groups, might was well verify that the ID matches here as well.

Reply:
In this particular place in the code there is not actual output Path, so we cannot test that. But I added it in a test above where we actually do have an output Path.